### ISSUE TITLE
Check and correct term definitions

### ISSUE THREAD

**Issue by cpavloud** [emo-bon/observatory-profile#4]
Check the [logsheet_schema_extended.csv](https://github.com/emo-bon/observatory-profile/blob/main/logsheet_schema_extended.csv) 

-- **review the definitions tab of the spreadsheets ([water](https://docs.google.com/spreadsheets/d/1jymTM_4L1T4hwRvSSbZQ7NoT6YfVZz0ClXHtSIel9w0/edit?gid=204464663#gid=204464663), [sediment](https://docs.google.com/spreadsheets/d/1T59_0877ZsnNBnmD65PP9uW87iUusrYY5iTPy0_RUnA/edit?gid=204464663#gid=204464663), [ARMS](https://docs.google.com/spreadsheets/d/1gGOmCKFP3LXlTGFRXDzWe4aNb8NDp_hRIyhCrFJkb5w/edit?gid=204464663#gid=204464663)) because sometimes the example does not follow the definition**. all definition tabs for Wa and all for So are the same (for each observatory), so you can review one Wa and one So, but if any corrections are necessary, they have to be done manually in all of them.

-- **check the official checklists** in the GSC website for the [water ](https://genomicsstandardsconsortium.github.io/mixs/0016014/) and the [sediment](https://genomicsstandardsconsortium.github.io/mixs/0016011/), as well as in the ENA website for the [water](https://www.ebi.ac.uk/ena/browser/view/ERC000024) and [sediment](https://www.ebi.ac.uk/ena/browser/view/ERC000021) **for the definition of those terms**
comm_samp
scientific_name
time_fi
tidal_stage
store_person
size_frac
ship_date
sampl_person
samp_mat_process
samp_size_mass
sample_collect_device
sample accession number
project accession number
or just accession number
project name
organization
all the loc_XX_XX in the logsheets
 
--  **fetch the URL of the terms**. 
E.g. the term "env_local_scale" has the URL https://genomicsstandardsconsortium.github.io/mixs/0000013/

-- **find terms in the BODC vocabulary**
biomass
n_alkanes
organism_count
samp_collect_device
samp_size_mass

-- **find ENVO or BODC or similar terms for** 
phytoplankton
diatoms
dinoflagellates
coccolitrophores
other flagellates

**Comment by cpavloud** [comment]
@09012000-tosca

**Comment by 09012000-tosca** [comment]
- comm_samp: def corresponds to the exemple. But there is no def in the GSC or ENA. 
- scientific_name: def DOES NOT corresponds to the exemple (marine plankton metagenome???). This term is missing in the seddiments logsheet (I do not know if is normal).
- time_fi: def corresponds to the exemple. But there is no def in the GSC or ENA.
- tidal_stage: def corresponds to the exemple and GSC or ENA.
- store_person: def corresponds to the exemple. But there is no def in the GSC or ENA.
- size_frac: def corresponds to the exemple. But there is no def in the GSC or ENA.
- ship_date: def corresponds to the exemple. But there is no def in the GSC or ENA.
- sampl_person:  def corresponds to the exemple.. In ARMS the definition is not correct "Name of person who sampled the seawater" 
- samp_mat_process: ok
- samp_size_mass: there is no this term in water and arms sheets. def corresponds to the exemple. probably because is samp size volume?. there is no def in the GSC or ENA.
- sample_collect_device:there is no in ARMS. Def corresponds to example. def is really similar to the ENA def. 
- sample accession number: there is the ENA accession number. It is okay or we need a sample accession number also for genoscope example? 
- project accession number or just accession number: there is the ENA accession number. It is okay or we need a sample accession number also for genoscope example? 
- project name: okay
- organization: okay


questions: alot of this terms definitions are quite specific on emobon. example scientific name we look at NCBI. Do we want an URl correspondant within NERC voc terms for this also if is not that specific?

**Comment by kmexter** [comment]
yes, a lot of the terms are specific to us. for example, a "person" can be defined not via BODC but schema.org as Person (see https://schema.org/Person), however a "sampling person" will never have a defintion as it is so specific, so for that one we do not expect a URL (if there is one, great, if not, that is fine). 
accession numbers: if you can find a term for "accession number" that is sufficient.

**Comment by 09012000-tosca** [comment]
- tidal_stage URI: [MIXS:0000750](https://w3id.org/mixs/0000750)
- project name URI  [MIXS:0000092](https://w3id.org/mixs/0000092)

**Comment by 09012000-tosca** [comment]
-- find terms in the BODC vocabulary

- biomass: 
http://vocab.nerc.ac.uk/collection/P02/current/FIBM/ it is specific to fish but is possible to make it more general: Parameters quantifying the mass in total or by species per unit area or per unit volume in any body of fresh or salt water expressed in any form (e.g. wet weight, dry weight, carbon, nitrogen, etc.)
 Other option: the one proposed in ENA vocabulary


- n_alkanes 
http://vocab.nerc.ac.uk/collection/P02/current/DALK/

- organism_count: no in the BODC but there is in the ENA voc and GSC [MIXS:0000103](https://w3id.org/mixs/0000103) 
- samp_collect_device no found
- samp_size_mass no found

**Comment by laurianvm** [comment]
Thanks a lot for the URI's of those term! :)

**Comment by 09012000-tosca** [comment]
Hi, I think the example should be changed :)

**Comment by cpavloud** [comment]
The scientific name definition is correct and the example is correct. 
It is just missing from the sediment logsheets.
And a Y should be added in columns MiXS_mandatory_(Y/N) and ENA_water_checklist_mandatory_(Y/N)

**Comment by kmexter** [comment]
>     * biomass:
>       http://vocab.nerc.ac.uk/collection/P02/current/FIBM/ it is specific to fish but is possible to make it more general: Parameters quantifying the mass in total or by species per unit area or per unit volume in any body of fresh or salt water expressed in any form (e.g. wet weight, dry weight, carbon, nitrogen, etc.)
>       Other option: the one proposed in ENA vocabulary
--> the BODC one does not work, @09012000-tosca can you give me the link to the ENA one?

**Comment by kmexter** [comment]
So as for the recommended changes to the definitions/examples - see comments from Tosca at the very top - that is for EMOBON HQ to do in the logsheets.
